apps/web/components/pages/task/details-section/blocks/task-secondary-info.tsx

```diff
@@ -179,34 +145,6 @@ const TaskSecondaryInfo = () => {
 				</ActiveTaskStatusDropdown>
 			</TaskRow>
 
-			{/* Task Labels */}
-			<TaskRow labelTitle={t('pages.taskDetails.LABELS')}>
```

Comment:
@CREDO23 why this whole block is removed?

Reply:
This one was duplicated as well @evereq 

![Screenshot from 2024-11-27 19-31-24](https://github.com/user-attachments/assets/b6a75369-9355-4ced-9dd3-68a3aee5a107)